src/ContainerHeader.jsx

```diff
@@ -32,7 +32,7 @@ class ContainerHeader extends React.Component {
 
     render() {
         return (
-            <div className="content-filter" >
+            <React.Fragment>
```

Comment:
This drops the content-filter class, but the next commit changes the CSS for this class. One of these two is inconsistent?

Reply:
There were two content-filter divs. I removed the inner one.